src/Microsoft.AspNetCore.Mvc.Razor.Extensions/DeclarationOnlyExtensionInitializer.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Language;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
```

Comment:
We should probably create the Version2_X shim before we start making 3.0 changes in the MVC shim.